src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/controllers/SaveRepresentativeClaimTest.java

```diff
@@ -89,8 +91,8 @@ public void setup() throws NotificationClientException {
         given(userService.generatePin(anyString(), anyString()))
             .willReturn(new GeneratePinResponse(PIN, LETTER_HOLDER_ID));
 
-        given(userService.getUserDetails(anyString()))
-            .willReturn(new UserDetails(CLAIMANT_ID, "claimant@email.com"));
+        given(userService.getUserDetails(anyString())).willReturn(
+            new UserDetails(CLAIMANT_ID, "claimant@email.com", SUBMITTER_FORENAME, SUBMITTER_SURNAME));
```

Comment:
why do you need the idam forename and surname?

Reply:
reprsentative names are not captured while adding claim, it is only with Idam while generating user

Reply:
The new builder can be used here.